Examples/Windpower Example/windpower-example.py

```diff
@@ -0,0 +1,85 @@
+import json
```

Comment:
Can you add what version of PySAM and SAM this was created with, and what it was last tested with?